.pre-commit-config.yaml

```
@@ -0,0 +1,199 @@
+              fi
+            done
+            echo "✅ Healthcare documentation validation passed"
+        files: \.(ts|tsx)$
+        pass_filenames: false
+
+# Global configuration
+default_install_hook_types: [pre-commit, pre-push]
+default_stages: [commit, push]
+
+# Performance optimization
+minimum_pre_commit_version: 3.0.0
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)